packages/client/package.json

```diff
@@ -83,7 +83,14 @@
       "import": "./wasm.js",
       "default": "./wasm.js"
     },
-    "./*": "./*"
+    "./*.js": "./*.js",
+    "./*.d.ts": "./*.d.ts",
+    "./*": {
+      "types": "./*.d.ts",
+      "require": "./*.js",
+      "import": "./*.js",
+      "default": "./*.js"
+    }
```

Comment:
This is to fix `https://github.com/prisma/prisma/issues/22875`. Also thanks to that, we can remove the `.js` extensions we just added on the types.